File: problems/3474/main.py
# 3474. 교수가 된 현우

import sys


def solve(n: int) -> int:
    ans = 0
    while n >= 5:
        n //= 5
        ans += n

    return ans


if __name__ == "__main__":
    t = int(sys.stdin.readline())

    for _ in range(t):
        print(solve(int(sys.stdin.readline())))

File: problems/3474/main.rs
// 3474. 교수가 된 현우

use std::io;

fn main() {
    let mut input = String::new();
    let stdin = io::stdin();

    stdin.read_line(&mut input).unwrap();
    let t: usize = input.trim().parse().unwrap();

    let mut lst = vec![];
    for _ in 0..t {
        input.clear();
        stdin.read_line(&mut input).unwrap();
        let n: u32 = input.trim().parse().unwrap();

        let (mut ans, mut div) = (0, 5);
        while div <= n {
            ans += n / div;
            div *= 5;
        }
        lst.push(ans.to_string())
    }

    println!("{}", lst.join("\n"))
}
